examples/ic_api/src/ic_api.ts

```diff
@@ -11,6 +12,26 @@ import {
     Update
 } from 'azle';
 
+// returns the argument data as bytes.
+export function arg_data_raw(
+    arg1: blob,
+    arg2: int8,
+    arg3: boolean,
+    arg4: string
+): Query<blob> {
+    return ic.arg_data_raw();
+}
+
+// returns the length of the argument data in bytes
+export function arg_data_raw_size(
```

Comment:
Let's add one record and one enum

Reply:
There are issues with records and enums. You can track this in https://github.com/demergent-labs/azle/issues/487.